locales/de/common.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "ImprintAndContact": "Impressum und Kontakt"
```

Comment:
see above and you should create a new language file Footer.json, see Search.json